app/models/host.rb

```diff
@@ -1,16 +1,10 @@
-$LOAD_PATH << File.join(GEMS_PENDING_ROOT, "util/xml")
-$LOAD_PATH << File.join(GEMS_PENDING_ROOT, "util/win32")
-$LOAD_PATH << File.join(GEMS_PENDING_ROOT, "metadata/linux")
```

Comment:
@jrafanie Not sure what to do about the untested code, at least it's early in the F-release.